cmd/state/main.go

```diff
@@ -246,6 +246,10 @@ func run(args []string, isInteractive bool, cfg *config.Instance, out output.Out
 		if childCmd != nil {
 			cmdName = childCmd.UseFull() + " "
 		}
+		nonInteractiveError := &prompt.NonInteractiveError{}
+		if errors.As(err, &nonInteractiveError) {
+			err = locale.WrapInputError(err, locale.T("err_non_interactive_terminal"))
+		}
```

Comment:
This feels way too specific to live in main.go. Let's just raise this in prompt?